src/lib/serviceRoutines/getAllEntitiesWithTypeAndId.cpp

```diff
@@ -39,6 +41,18 @@
 /* ****************************************************************************
 *
 * getAllEntitiesWithTypeAndId - 
+*
+* GET /v1/contextEntities/type/{entity::type}/id/{entity::id}
+* GET /ngsi10/contextEntities/type/{entity::type}/id/{entity::id}
+*
+* Payload In:  None
+* Payload Out: ContextElementResponse
+* 
+* URI parameters:
+*   - attributesFormat=object
```

Comment:
In this case limit/offset/details don't make sense, do they? Almost sure at 99% percent, just to be sure...